docs/source/migrating.rst

```diff
@@ -0,0 +1,189 @@
+Migrating from IPython
+======================
+
+`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+has changed a few things, moving the various language-agnostict
+components of IPython under the Jupyter umbrella. This doc should help
```

Comment:
"This document describes what has changed, and how you will need to modify your code and configuration."